base/gmp.jl

```diff
@@ -458,7 +475,7 @@ powermod(x::Integer, p::Integer, m::BigInt) = powermod(big(x), big(p), m)
 
 function gcdx(a::BigInt, b::BigInt)
     if b == 0 # shortcut this to ensure consistent results with gcdx(a,b)
-        return a < 0 ? (-a,-one(BigInt),zero(BigInt)) : (a,one(BigInt),zero(BigInt))
+        return a < 0 ? (-a,-ONE,ZERO) : (a,ONE,ZERO)
```

Comment:
It has nothing to do with aliasing.

If I do `a,b,c = gcdx(BigInt(1), BigInt(0))` and I do `f(b)` which mutates `b` (which is not forbidden, since it is mutable), then the global variable `ONE` will be changed.

Again, this has nothing to do with aliasing.